Cargo.toml

```
@@ -8,9 +8,13 @@ description = "Unofficial rust client for the [V7 annotation platform](https://d
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = "1.0.69"
-serde = "1.0"
+anyhow = "1.0"
+reqwest = { version = "0.11", features = ["json"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.93"
```

```suggestion
serde_json = "1.0"
```

**question:** To stick with the format of the other versions, can we drop the build number from `serde_json`?

---

yep that slipped through the cracks